src/fastertransformer/triton_backend/llama/LlamaTritonModel.cc

```diff
@@ -41,13 +41,6 @@ std::shared_ptr<AbstractTransformerModel> AbstractTransformerModel::createLlamaM
             reader.GetInteger("ft_instance_hyperparameter", "enable_custom_all_reduce", 0),
             model_dir);
     }
-    if (data_type == "bf16") {
```

Comment:
Why do we revert bf16 support?

Reply:
some kernels are not properly configured during testing, will take a deeper look and add it back

Reply:
@yinghai , bf16 is back 😄